commons/src/main/scala/com/lynbrookrobotics/potassium/commons/lift/offloaded/OffloadedLift.scala

```diff
@@ -0,0 +1,52 @@
+package com.lynbrookrobotics.potassium.commons.lift.offloaded
+
+import com.lynbrookrobotics.potassium.Signal
+import com.lynbrookrobotics.potassium.commons.lift.Lift
+import com.lynbrookrobotics.potassium.control.offload.EscConfig.{forwardToAngularPositionGains}
+import com.lynbrookrobotics.potassium.control.offload.OffloadedSignal.{OpenLoop, PositionBangBang, PositionPID}
+import com.lynbrookrobotics.potassium.control.offload.{EscConfig, OffloadedSignal}
+import com.lynbrookrobotics.potassium.streams._
+import squants.Dimensionless
+import squants.space.Length
+
+abstract class OffloadedLift extends Lift {
+  override type LiftSignal = OffloadedSignal
+  override type Properties <: OffloadedLiftProperties
+
+  override def positionControl(target: Stream[Length])
+                              (implicit properties: Signal[Properties],
+                               hardware: Hardware): (Stream[Length], Stream[LiftSignal]) = (
+    hardware.
+      position.
+      zipAsync(target)
+      .map(t => t._2 - t._1),
+
+    target.map { it =>
+      val curProps = properties.get
+      PositionPID(
+        forwardToAngularPositionGains(curProps.positionGains)(curProps.escConfig), curProps.toNative(it)
+      )
+    }
+  )
+
+  override def stayAbove(target: Stream[Length])
+                        (implicit p: Signal[Properties],
+                         hardware: Hardware): Stream[LiftSignal] = target.map { it =>
+    PositionBangBang(
+      forwardWhenBelow = true, reverseWhenAbove = false,
+      signal = p.get.toNative(it)
```

Comment:
I think this abstraction is fine, especially since in the future we may be dealing with sensors other than potentiometers. Do you think it would be possible though to have a conversion method (in the frc module) that handles converting voltage to the raw values reported by the Talon SRX?